vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -111,21 +111,25 @@ thread_local std::int32_t FlyoutShadowNode::s_cOpenFlyouts = 0;
 FlyoutShadowNode::~FlyoutShadowNode() {
   m_touchEventHanadler->RemoveTouchHandlers();
   m_previewKeyboardEventHandlerOnRoot->unhook();
+  m_windowedPopupChildView = nullptr;
 }
 
 void FlyoutShadowNode::AddView(ShadowNode &child, int64_t index) {
   auto childView = static_cast<ShadowNodeBase &>(child).GetView();
   m_touchEventHanadler->AddTouchHandlers(childView);
   m_previewKeyboardEventHandlerOnRoot->hook(childView);
 
-  if (m_flyout != nullptr)
+  if (m_flyout != nullptr) {
     m_flyout.Content(childView.as<winrt::UIElement>());
+    m_windowedPopupChildView = m_flyout.Content().as<XamlView>();
```

Comment:
>m_flyout.Content().as<XamlView>() [](start = 31, length = 33)

Can this be m_windowedPopupChildView = ChildView?

Reply:
I'll try that.

---
In reply to: [304624011](https://github.com/microsoft/react-native-windows/pull/2781#discussion_r304624011) [](ancestors = 304624011)